### ISSUE TITLE
Appium Inspector - Choose to remove prefixes when saving a recorded file (can't run them locally)

### ISSUE THREAD

**Issue by SuperRoach** [appium/appium-inspector#643]
## Appium or Appium Desktop?

Appium inspector

- [x] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

When using a test script generated locally by Appium inspector, The usage of the appium: prefixes to capabilities makes the scripts unusable locally. 

I can make a script to find and remove instances where it gets added into the capability ( aka caps) section of the JS, and I'm not sure if it applies to other parts.

## Environment

- I am running Appium Desktop version _<insert version here>_.
- I am on (pick one):
    - [x] Mac
    - [ ] Windows
    - [ ] Linux

With the current state of making/launching an app on a mobile device, the capabilities that you specify require you to tick the box that says "Add prefix to capabilities". If I don't check that when I'm running locally, I get an error which I'll have to check - I think it was either before starting saying "cloud instances require this option", or something else.

If I was able to do that (launch a saved set without prefixes) locally though, I feel it would save the trouble of later on when generating a script it would not append the appium: prefixes in the first place. 

But it might be nice to have an option near "add boiler plate" to choose to strip them out, which would cover both use cases I guess. Not sure what your thoughts are on it?

**Comment by jlipps** [comment]
I don't think I understand the problem. Are you saying that the same set of capabilities doesn't work on your local server as on a remote (cloud) server? In that case I would contact the cloud provider's support team and ask them why they don't support W3C-style capabilities.

As a user right now, I would probably keep different sets of capabilities for local vs cloud if that's the case.

But maybe you're saying the problem is something else?

**Comment by SuperRoach** [comment]
I come bearing screenshots! So in short, if I try to launch a webdriver config, if I have the prefix on them, it will not be recognised by the server.

Stealing a screenshot from another is here: https://user-images.githubusercontent.com/32598251/200779722-47e7fb41-c6c8-4999-85d5-e190d03b79c8.png. You can see that it has added the prefixes to it - That will launch fine.

![image](https://user-images.githubusercontent.com/63333/200837375-a311749f-dbf5-4e55-bc28-b78e1231f369.png)
Here is a screenshot of settings, and I have the Add prefixes on Start deselected. 

On doing this, this is the error message I get:
![image](https://user-images.githubusercontent.com/63333/200837839-b74f5ad0-ee87-455f-879b-3d2ec727a3ba.png)
With Add Prefixes selected, this works fine.

Because of adding the prefixes in however, if I follow through this flow (launch the app with those prefixes), When I record a test, then save it with the boilerplate/capabilities attached, it will also have the appium: prefixes there. On my setup at least, this means that the .js file I saved will not work.

I hope that this helped clarify it for you!

**Comment by jlipps** [comment]
What error do you get from the server when you do include the prefixes? Is it some cloud server?

**Comment by brett-james-rocketlab** [comment]
It's not a cloud server, it's appium running on my local computer. If I do it again and have that error pop up, there's no output in the server console (Appium Server GUI)

**Comment by brett-james-rocketlab** [comment]
Ok now we are getting somewhere, so here is two capabilities, one I've fixed up, and one direct from the recorder with show boilerplate on.


`// javascript (fixed caps with prefixes manually removed)
// const capabilities = {
//   path: "/wd/hub",
//   port: 4723,
//   capabilities: {
//     platformName: "Android",
//     appium:platformVersion: "12",
//     appium:deviceName: "R5CR10E0ZGF",
//     implicitWait: 10,
//     autoGrantPermissions: true,
//     app: "/Users/brett/development/adultmatchmaker/android/app/build/outputs/apk/release/app-release.apk",
//     appPackage: "com.adultmatchmaker.messenger",
//     appActivity: ".MainActivity",
//     automationName: "UiAutomator2",
//     screenShotOnError: true,
//   },
// };

// This one is a non working one, direct from Appium Inspecter after having to check add prefixes when launching this profile/set
const capabilities = {"platformName":"Android","appium:platformVersion":"12","appium:deviceName":"R5CR10E0ZGF","appium:autoGrantPermissions":true,"appium:app":"/Users/brett/development/quantaco/android/app/build/outputs/apk/release/app-release.apk","appium:appPackage":"co.quantaco","appium:appActivity":".MainActivity","appium:automationName":"UiAutomator2","appium:ensureWebviewsHavePages":true,"appium:nativeWebScreenshot":true,"appium:newCommandTimeout":3600,"appium:connectHardwareKeyboard":true};`

brett@Bretts-MBP appiumtesting % node amm-login.js
```
/Users/brett/development/rocketlab/appiumtesting/node_modules/@wdio/config/build/utils.js:55
            throw new Error(`Required option "${name.toString()}" is missing`);
```
I get the above error

**Comment by jlipps** [comment]
I'm still confused, are you saying it's the JS generated by the boilerplate that results in an error?

In any case I would not recommend using the Appium Server GUI. It's not being maintained and Appium 1.x is no longer supported. There's no guarantee that the Inspector will continue to interoperate with it.

**Comment by brett-james-rocketlab** [comment]
I think I'm saying that the JS generated by the boilerplate results in an error when I try to run it via Appium Server (that uses 1.x), with the error generated above. I didn't know there is a v2 though, would that let it work if you know?

**Comment by jlipps** [comment]
@brett-james-rocketlab yes, that documentation has unfortunately not been updated to reflect reality. it will be deleted soon. the new documentation is currently hosted at https://appium.github.io/appium/docs/en/2.0/.

the code the inspector generates is now targeting appium 2, yes. so i wouldn't expect it to work with a 1.x server, since appium 1.x is no longer supported.

**Comment by eglitise** [comment]
Closing this issue. Appium Server GUI is no longer supported, and the current documentation at https://appium.io/ points to Appium 2 for local setup, which will not have this problem.
